server/server.go

```diff
@@ -99,8 +97,12 @@ func WithLogging(
 func (svr *Server) Start() error {
 	mux := http.NewServeMux()
 
-	mux.HandleFunc(GetRoute, WithLogging(WithMetrics(svr.HandleGet, svr.m), svr.log))
-	mux.HandleFunc(PutRoute, WithLogging(WithMetrics(svr.HandlePut, svr.m), svr.log))
+	mux.HandleFunc("/get/", WithLogging(WithMetrics(svr.HandleGet, svr.m), svr.log))
+	// we need to handle both: see https://github.com/ethereum-optimism/optimism/pull/12081
+	// /put is for generic commitments, and /put/ for keccak256 commitments
+	// TODO: we should probably separate their handlers?
+	mux.HandleFunc("/put", WithLogging(WithMetrics(svr.HandlePut, svr.m), svr.log))
+	mux.HandleFunc("/put/", WithLogging(WithMetrics(svr.HandlePut, svr.m), svr.log))
```

Comment:
@epociask can you review that this comment and the routes make sense. tests are passing but still want to be 100% certain since I still don't get our routing logic with the different paths. Should probably document this api at some point..